charts/castai-kvisor/templates/secret.yaml

```diff
@@ -14,3 +14,8 @@ metadata:
 data:
   API_KEY: {{ required "castai.apiKey must be provided" .Values.castai.apiKey | b64enc | quote }}
 {{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.castai.certsSecret }}
```

Comment:
Shouldn't it be applied to helm chart release namespace?

Reply:
Helm doesn't support such feature. You need to explicitly add `namespace: {{ .Release.Namespace }}` as we do for all other resources.

Reply:
I'm pretty sure that if you are not defining namespace in some object, Helm adds .Release.Namespace to that object. But I will add namespace to close this